code/modules/antagonists/heretic/items/madness_mask.dm

```diff
@@ -58,7 +58,7 @@
 		if(human_in_range.is_blind())
 			continue
 
-		SEND_SIGNAL(human_in_range, COMSIG_HERETIC_MASK_ACT, rand(-2, -20) * delta_time)
+		human_in_range.mob_mood.direct_sanity_drain(rand(-2, -20) * delta_time)
```

Comment:
Any chance you could fix #69082 by making this a mood event?  If not I can attempt this later once this PR gets merged.

Reply:
this has different behaviour than a mood event so i think its a little out of scope

Reply:
Understandable